shell/src/main/java/alluxio/shell/command/MountCommand.java

```diff
@@ -92,14 +128,27 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
       options.setShared(true);
     }
 
+    if (cl.hasOption("option")) {
+      Map<String, String> properties = new HashMap<>();
+      String[] optionValues = cl.getOptionValues("option");
+      for (String option : optionValues) {
+        Matcher matcher = OPTION_PATTERN.matcher(option);
+        Preconditions.checkArgument(matcher.matches(), "Unrecognized property %s", option);
+        String key = matcher.group(1);
+        String value = matcher.group(2);
+        properties.put(key, value);
+      }
+      options.setProperties(properties);
+    }
+
     mFileSystem.mount(alluxioPath, ufsPath, options);
     System.out.println("Mounted " + ufsPath + " at " + alluxioPath);
     return 0;
   }
 
   @Override
   public String getUsage() {
-    return "mount [-readonly] [-shared] [-P <properties file name>] <alluxioPath> <ufsURI>";
+    return "mount [--readonly] [--shared] [--option <key=val>] [-P <path>] <alluxioPath> <ufsURI>";
```

Comment:
removed. After fixing `bin/alluxio`, we could use `--option` to pass properties with whitespace now. therefore `-P` is no more needed